gravitee-apim-distribution/pom.xml

```diff
@@ -46,7 +46,7 @@
     <properties>
         <!-- Versions of the plugins for the full distribution -->
         <!-- Management API & Gateway -->
-        <gravitee-connector-http.version>1.1.4</gravitee-connector-http.version>
+        <gravitee-connector-http.version>1.1.5-7750-websocket-headers-SNAPSHOT</gravitee-connector-http.version>
```

Comment:
Is this expected?

Reply:
it was temporary to check the build in the CI. It has been fixed by https://github.com/gravitee-io/gravitee-api-management/commit/382c11d91b05ea25203ac94d8450e303a96dc6cc